src/pages/gateway/developer-tools.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Developer tools
+description: Learn how to use development tools to create a local environment and use environment variables with API Mesh.
+---
+
+# Developer tools
+
+The processes covered in this topic allow developers to set up a local environment, use environment variables, and directly reference files in API Mesh for Adobe Developer App Builder.
+
+## Initiate a local environment
+
+A local development environment for API Mesh for Adobe Developer App Builder allows you to run a local version for development and testing purposes.
+
+The [`aio api-mesh:init` command](./command-reference.md#aio-api-meshinit) allows you to build a local development environment at the specified location.
+
+1. Run the following command.
+
+    ```terminal
+    aio api-mesh:init <project-name>
+    ```
+
+1. Select the folder to install the dev environment in.
+
+1. Indicate if you want to use `git`. (Requires [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).)
```

Comment:
Are git and npm/yarn expected to be mutually exclusive? If you specify Yes to the `git` question, does the `npm` question still appear?

Reply:
Yes, it is my understanding that the local environment can only use `npm` OR `yarn`. I believe `git` is entirely unrelated from this choice. @revanth0212 can you confirm?

Reply:
The question behind the original question: When would the developer say no to both git and package manager? Or say yes to both?

Reply:
i've clarified some of the language here to address the confusion. saying `yes` to `git` indicates that you want the directory to be a `git` project. regarding the package manager prompt, there is no option to say `no`, the user must choose either `npm` or `yarn`.